tox.ini

```diff
@@ -16,6 +17,10 @@ commands =
     # NB: New projects should consider replacing the above with below!
     # ruff check . {posargs}
 
+    # below are formatter checks
+    ruff format . --check {posargs}
+    ruff check . --select  I {posargs}
```

Comment:
For this could just add `--select I` to the rule on line 16?

Reply:
Ya we could, I  kept it separate because we didn't discuss import sorting and I wanted to delineate where we were getting that from vs the current implementation. I'll change it, if we are ok with using the linter for this rather than a stand alone package like `isort`